lib/ret_web/controllers/api-internal/v1/presence_controller.ex

```diff
@@ -1,10 +1,23 @@
 defmodule RetWeb.ApiInternal.V1.PresenceController do
   use RetWeb, :controller
+  alias Ret.NodeStat
 
   # Get presence count
   def show(conn, _) do
     count = RetWeb.Presence.present_ccu_in_room_count()
 
     conn |> send_resp(200, %{count: count} |> Poison.encode!())
   end
+
+  # Params start_time and end_time should be in iso format such as "2000-02-28 23:00:13"
+  # or what is returned from NaiveDateTime.to_string()
+  def daily_max(conn, %{"start_time" => start_time_str, "end_time" => end_time_str}) do
```

Comment:
Should this be in presence controller or in a new node stat controller?

Reply:
Nah, I think it's fine to keep it here.